File: app/src/main/java/com/rkbapps/imagesearch/util/ApiData.java
package com.rkbapps.imagesearch.util;

public class ApiData {

    public static final String API_KEY = "x9Yf1QT8KXSpwH4ILI0smrHAIGY7BwFarViZrYM8jxlY2vABSWIOyDX5";

    public static String getUrl(String query, Integer page) {
        return "https://api.pexels.com/v1/search/?page=" + page + "&per_page=30&query=" + query;
    }

    public static final String BASE_URL = "https://api.pexels.com/v1/";



}

File: app/src/main/java/com/rkbapps/imagesearch/util/ApiData.kt
package com.rkbapps.imagesearch.util

import com.rkbapps.imagesearch.BuildConfig


object ApiData {
    const val API_KEY = BuildConfig.API_KEY
    fun getUrl(query: String, page: Int): String {
        return "https://api.pexels.com/v1/search/?page=$page&per_page=30&query=$query"
    }
    const val BASE_URL = "https://api.pexels.com/v1/"
}